magicbot/magicrobot.py

```diff
@@ -386,6 +396,7 @@ def _create_components(self):
 
             # Automatically inject a logger object
             component.logger = logging.getLogger(m)
+            component.logger.addFilter(PeriodicFilter(self))
```

Comment:
Is it a good idea if we make them add it themselves? And the structure is all in place in magicbot, you just have to enable it?